packages/@aws-cdk/core/lib/asset-staging.ts

```diff
@@ -145,15 +164,44 @@ export class AssetStaging extends Construct {
     }
   }
 
-  private bundle(options: BundlingOptions): string {
-    // Temp staging directory in the working directory
-    const stagingTmp = path.join('.', STAGING_TMP);
-    fs.ensureDirSync(stagingTmp);
+  private getAssetRelativePath(assetHash: string, extension = '') {
```

Comment:
```suggestion
  private renderAssetFilename(assetHash: string, extension = '') {
```

Reply:
I renamed this function and pulled it out into a local floating function.